src/app/main/modals/tooltip/tooltip.component.html

```diff
@@ -3,15 +3,8 @@
 
     <example-comp #ex comp-name="TOOLTIP" fxFill
       (showSource)="onShowSource(ex, tooltipInput.value, matSelect.value, showDelayInput.value, hideDelayInput.value)">
-      <o-form editable-detail="no" show-header="no" layout-direction="row">
```

Comment:
The o-form cannot be deleted, it is necessary that any input be inside an o-form